lib/agenda.js

```diff
@@ -420,33 +420,26 @@ Agenda.prototype._findAndLockNextJob = function(jobName, definition, cb) {
   var self = this,
       now = new Date(),
       lockDeadline = new Date(Date.now().valueOf() - definition.lockLifetime);
-
-  // Don't try and access Mongo Db if we've lost connection to it. Also see clibu_automation.js db.on.close code. NF 29/04/2015
-  // Trying to resolve crash on Dev PC when it resumes from sleep.
   var s = this._mdb.s || this._mdb.db.s;
-  if (s.topology.connections().length === 0) {
```

Comment:
I think we had trouble with the scheduler restarting when we took this away. It seemed to work best when we just stopped throwing the error, though I don't know why